README.md

```diff
@@ -33,43 +40,132 @@ This middleware avoids this type of queries checking if the incoming query is wh
 
 # Installation
 
-`npm install --save graphql-query-whitelisting`
+`npm install --save graphql-query-whitelist graphql body-parser`
 
 In your app:
 
 ```js
 import express from 'express'
 import bodyParser from 'body-parser'
-import Redis from 'ioredis'
+import graphqlWhitelist, { MemoryStore } from 'graphql-query-whitelist'
 
 const app = express()
-const redis = new Redis()
+const store = new MemoryStore()
+// body-parser must be included before including the query whitelist middleware
+app.use(bodyParser.json())
+app.post('/graphql', graphqlWhitelist({ store }))
+```
+
+Before each request is processed by GraphQL, it will check if the inbound query is in the whitelist or not.
+If it's not in the whitelist, it will respond with a 401 status code.
+
+# Running queries only sending the queryId
+
+Since the server has access to the query store, and the store has access to the full queries, it's possible to run a query just sending the queryId.
```

Comment:
Esto nunca me queda del todo claro: no debería ser _just **by** sending the queryId_?